src/lsm/table_memory.zig

```diff
@@ -237,8 +237,9 @@ pub fn TableMemoryType(comptime Table: type) type {
             var source_index: u32 = offset;
             var target_index: u32 = offset;
             while (source_index < source_count) {
-                const value = values[source_index];
-                values[target_index] = value;
```

Comment:
Surprised this helps!

Reply:
I wonder if it makes to even pull this out in a separate loop that finds first mismatch?